### PR TITLE
add more docs about scatter/gather

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -619,6 +619,32 @@ For mapping, sarek follows the parameter suggestions provided in this [paper](ht
 
 In addition, currently the mismatch penalty for reads with tumor status in the sample sheet are mapped with a mismatch penalty of `-B 3`.
 
+## How to manage scatter/gathering (parallelization with-in each sample)
+
+While nextflow ensures all samples are run in parallel, the pipeline in addition attempts to split input files for one sample into smaller chunks and processes these in parallel.
```
Comment (maxulysse) [inline docs/usage.md:624]:
```suggestion
While Nextflow ensures all samples are run in parallel, the pipeline can split input files for each sample into smaller chunks which are processes in parallel.
```

File: docs/usage.md [hunk 2]
```diff
@@ -619,6 +619,32 @@ For mapping, sarek follows the parameter suggestions provided in this [paper](ht
...
+This allows on the one hand to speed up analysis for individual chunks, but might occupy more storage space.
```
Comment (maxulysse) [inline docs/usage.md:625]:
```suggestion
This speeds up analysis for individual chunks, but might occupy more storage space.
```

File: docs/usage.md [hunk 3]
```diff
@@ -619,6 +619,32 @@ For mapping, sarek follows the parameter suggestions provided in this [paper](ht
...
+
+Therefore, the different scatter/gather options can be set by the user:
+
+### Split Fastq files
+
+By default, the input fastq files are split into smaller chunks with FASTP, mapped in parallel, and then merged and duplicate marked. This can be customized by setting the parameter `--split_fastq`.
+This parameter determines how many reads are within each split. Setting it to `0` will turn of any splitting and only one mapping process is run per input fastq file.
+
+> FastP always creates as many chunks as CPUs are specified (by default 12). Thus, the parameter `--split_fastq` is an upper bound, e.g. if 1/12th of the Fastq file exceeds the provided value another fastq file will be generated.
```
Comment (maxulysse) [inline docs/usage.md:634]:
```suggestion
> **NB:** FASTP can only create as many chunks as CPUs are specified (by default 12). Thus, the parameter `--split_fastq` has an upper bound, e.g. if 1/12th of the fastq file exceeds the provided value another fastq file will be generated.
```

File: docs/usage.md [hunk 4]
```diff
@@ -619,6 +619,32 @@ For mapping, sarek follows the parameter suggestions provided in this [paper](ht
...
+
+### Intervals for Base Quality Score Recalibration and Variantcalling
+
+The pipeline attempts to parallelize base quality score recalibration and variant calling where possible across genomic chunks of roughly similar sizes of one samples. For this, a bed file with interesting genomic regions is used. By default, the intervals file for WGS provided by GATK is used (details [here](https://gatk.broadinstitute.org/hc/en-us/articles/360035889551-When-should-I-restrict-my-analysis-to-specific-intervals-)). When running targeted analysis, it is usually good to use the bed file containing the targeted regions.
```
Comment (maxulysse) [inline docs/usage.md:638]:
```suggestion
The pipeline can parallelize base quality score recalibration and variant calling across genomic chunks of roughly similar sizes.
For this, a bed file containing genomic regions of interest is used, it's the intervals file.
By default, the intervals file for WGS used is the one provided by GATK (details [here](https://gatk.broadinstitute.org/hc/en-us/articles/360035889551-When-should-I-restrict-my-analysis-to-specific-intervals-)).
When running targeted analysis, it is recommended to use the bed file containing the targeted regions.
```

File: docs/usage.md [hunk 5]
```diff
@@ -619,6 +619,32 @@ For mapping, sarek follows the parameter suggestions provided in this [paper](ht
...
+
+The amount of scatter/gathering can be customized by adjusting the parameter `--nucleotides_per_second`.
+
+> **NB:** The _same_ intervals are processed regardless of the number of groups. The number of groups however determines over how many compute nodes the analysis is scattered.
```
Comment (maxulysse) [inline docs/usage.md:642]:
```suggestion
> **NB:** The _same_ intervals are processed regardless of the number of groups. The number of groups however determines over how many compute nodes the analysis is scattered on.
```

**Review by maxulysse — APPROVED** [review]

**Comment by FriederikeHanssen** [comment]
@drpatelh if I could bother you for a quick look. Is this a bit more clear now in your opinion?

**Review by maxulysse — APPROVED** [review]

**Comment by FriederikeHanssen** [comment]
@nf-core-bot fix linting

**Pull request merged**